src/js-executor/execute-es-module.js

```diff
@@ -0,0 +1,50 @@
+export async function executeWasmCode(container, jsCode, wasmCode) {
+  const newCode = prepareJsCode(jsCode);
+  return execute(container, newCode, wasmCode);
+}
+
+export async function executeWasmCodeWithSkiko(container, jsCode, wasmCode) {
+  const skikoMjs = new URL("../../skiko/skiko.mjs", import.meta.url).href;
+  const skikoWasm = new URL("../../skiko/skiko.wasm", import.meta.url).href;
+  const skikoCode = (await (await fetch(skikoMjs)).text())
```

Comment:
It's look like skiko is fetching every time when `executeWasmCodeWithSkiko` is called. Is it possible to somehow check if it was already loaded?

Reply:
Done